### PR TITLE
Annual Contributions higher amounts test

### PR DESCRIPTION
**Pull request by jranks123**
## Why are you doing this?
This PR launches a new round of annual contributions testing, where we add a variant with higher suggested amounts for annual contributions. 

For the control, there is no annual contributions tab, so the page is unchanged. Here is the AU landing page as an example:

| Control |
|------|
|https://github.com/guardian/support-frontend/pull/888#issue-209763716
image|

For the annual variants, when you hit the landing page monthly is still the default, but there is an additional `Annually` tab, which looks like this:

### UK
|Annual|Annual Higher|
|-----|------|
|ukannual|ukannualhigher|

### US
|Annual|Annual Higher|
|-----|------|
|usannual|usannualhigher|

### Australia
|Annual|Annual Higher|
|-----|------|
|auannual|auannualhigher|

### New Zealand
|Annual|Annual Higher|
|-----|------|
|nzannyal|nzannualhigher|

### Canada
|Annual|Annual Higher|
|-----|------|
|caannual|caannualhigher|

### EU
|Annual|Annual Higher|
|-----|------|
|euannual|euannualhigher|

### International
|Annual|Annual Higher|
|-----|------|
|intannual|intannualhigher|

### CODE REVIEW

File: assets/helpers/contributions.js [hunk 1]
```diff
@@ -62,6 +62,7 @@ const numbersInWords = {
   '166': 'one hundred and sixty six',
   '250': 'two hundred and fifty',
   '500': 'five hundred',
+  '750': 'seven hundred',
```
Comment (tsop14) [inline assets/helpers/contributions.js]: 'seven hundred and fifty'

File: assets/components/contributionSelection/contributionSelectionReducer.js [hunk 2]
```diff
@@ -154,7 +152,11 @@ function contributionSelectionReducerFor(scope: string, countryGroupId: CountryG
         };
 
       case 'SET_AMOUNT':
-        return updatePredefinedAmount(state, action.amount);
+        return updatePredefinedAmount(state, state.contributionType, action.amount);
```
Comment (Mullefa) [inline assets/components/contributionSelection/contributionSelectionReducer.js]: any reason why we're using `state.contributionType ` here and `action.contributionType` for the next case?

File: assets/components/contributionSelection/contributionSelectionReducer.js [hunk 3]
```diff
@@ -154,7 +152,11 @@ function contributionSelectionReducerFor(scope: string, countryGroupId: CountryG
...
+
+      case 'SET_AMOUNT_FOR_CONTRIBUTION_TYPE':
+        return updatePredefinedAmount(state, action.contributionType, action.amount);
```
Comment (Mullefa) [inline assets/components/contributionSelection/contributionSelectionReducer.js]: ... the next case

File: assets/helpers/contributions.js (hunk 1 above)
Comment (jranks123) [reply assets/helpers/contributions.js]: good spot!

File: assets/components/contributionSelection/contributionSelectionReducer.js (hunk 3 above)
Comment (jranks123) [reply assets/components/contributionSelection/contributionSelectionReducer.js]: The difference is whether you are updating the selected amount of the currently selected contribution type (`state.contributionType`) or the selected amount of a contribution type of your choosing (`action.contributionType`).

**Pull request merged**